src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 
+// will be able to set either height or width of component but not both:w
+const WIDTH_HEIGHT_RATIO = 1.583;
+const height = 300;
+
 export default function VeteranIDCard() {
+  // <img className="example-photo" alt="placeholder" src="/img/air-force-coat-of-arms.png"/>
   return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
+    <div style={{ marginTop: '50px', backgroundColor: 'grey', height, width: height * WIDTH_HEIGHT_RATIO }}>
+      <svg width={'100%'} height={'100%'}>
+        <image width={'100%'} href={'/img/vic-preview-background.svg'} />
+        <image height={'28.5%'} href={'/img/va-seal.png'} x={'6.5%'} y={'7.3%'} />
+        <text x={'26.3%'} y={'24.3%'} stroke={'none'} fill={'white'} fontFamily={'Arial, Helvetica, sans-serif'} fontSize={23}>
+          Veteran Identification Card
+        </text>
+        <image height={'51.8%'} href={'/img/example-photo-2.png'} x={'62.1%'} y={'34.1%'} />
+        <image height={'21.1%'} href={'/img/air-force-coat-of-arms.png'} x={'7.3%'} y={'70%'} />
+        <text x={'7.3%'} y={'55.2%'} stroke={'none'} fill={'white'} fontFamily={'Arial, Helvetica, sans-serif'} fontSize={23}>
+          Firstname Lastname
```

Comment:
Open to ideas on how to detect breaking this into two lines. Would like to stick to using `<text>` svg elements but that could get tricky to do- maybe measuring text width with a canvas.

Reply:
Could we use number of characters?

Reply:
We could with a monospaced font. There's a pretty big width difference between 
WWW and 
lll

I know we can render the text to a canvas and measure the width but it might be apples and oranges. There's an package for [word wrapping](https://www.npmjs.com/package/word-wrapper) in the wild or we could just make our own since this use case is very limited. 

We could go further: bump the font size down and measure recursively if the two line name doesn't fit the width constraint or support three line names if someone has a long hyphenated last name. 

Might be helpful to have some actual long last names to test with.